lib/iris/fileformats/netcdf/loader.py

```diff
@@ -236,7 +237,19 @@ def _get_cf_var_data(cf_var, filename):
             )
             # Get the chunking specified for the variable : this is either a shape, or
             # maybe the string "contiguous".
+            if CHUNK_CONTROL.mode == CHUNK_CONTROL.Modes.AS_DASK:
+                return as_lazy_data(proxy, chunks=None, as_dask=True)
```

Comment:
We typically avoid mutiple `return` statements at different points in a function/method, as it can be harder to follow. I would find it more readable if there is just the one call to `as_lazy_data()` at the end of this function, even though that requires `as_dask` to ALWAYS have a value (`False`/`True`).